asciinema2gif

```diff
@@ -88,7 +88,7 @@ done
 if [[ -z "${1}" ]] || [[ -n "${2}" ]]; then
   usage
   exit 1
-elif [[ "${1}" =~ ^[[:digit:]]+$ ]]; then
+elif [[ "${1}" =~ ^[[:alnum:]]+$ ]] && [[ $(curl -s -I "${asciinema_api}/${1}" | grep -c 200) == 2 ]]; then
```

Comment:
Should be `-eq 2` instead of `== 2`, for consistency with the rest of the script (see [line 24](https://github.com/jfmherokiller/asciinema2gif/blob/9f45aa39e4f96c819cff47956c23959cb7b7c2b7/asciinema2gif#L24)).